docs/src/main/asciidoc/spring-cloud-aws.adoc

```diff
@@ -201,7 +201,7 @@ errors if the properties are not configured at all.
 The Parameter Store and Secrets Manager Configuration support uses a bootstrap context to configure a default `AWSSimpleSystemsManagement`
 client, which uses a `com.amazonaws.auth.DefaultAWSCredentialsProviderChain` and `com.amazonaws.regions.DefaultAwsRegionProviderChain`.
 If you want to override this, then you need to
-https://cloud.spring.io/spring-cloud-static/Edgware.SR2/multi/multi__spring_cloud_context_application_context_services.html#_customizing_the_bootstrap_configuration[define your own Spring Cloud bootstrap configuration class]
+https://projects.spring.io/spring-cloud/spring-cloud.html#_customizing_the_bootstrap_configuration[define your own Spring Cloud bootstrap configuration class]
```

Comment:
Even more up to date links is: https://docs.spring.io/spring-cloud-commons/docs/current/reference/html/#customizing-the-bootstrap-configuration

Reply:
Updated. There are too many different ways to access various versions of the documentation.